ansible/tasks/nginx.yml

```diff
@@ -151,11 +148,9 @@
     --webroot
     --webroot-path /var/www/acme-challenges
     --domain {{ item.name }}
-    {% if 'staging' not in group_names %}
     {% for hostname in item.alternative_names %}
     --domain {{ hostname }}
     {% endfor %}
-    {% endif %}
```

Comment:
This will not work, we still need this check here or the staging server will try to get certs for websites we only have a live environment for like `execut.nl`. Keeping this check should be fine (so no alternative names when on the staging server)